python/tvm/relay/frontend/pytorch.py

```diff
@@ -595,15 +601,18 @@ def _impl(inputs, input_types):
         return _op.log(_op.tensor.sigmoid(data))
     return _impl
 
-def _adaptive_avg_pool_2d():
+def _adaptive_avg_pool_2d(prelude):
     def _impl(inputs, input_types):
         data = inputs[0]
         output_size = _infer_shape(inputs[1])
 
         def func(x):
             return _op.nn.adaptive_avg_pool2d(x, output_size=output_size)
 
-        if input_types[0] == "quint8":
+        # If a quantized Torch module is saved and loaded back, dtype will be dropped
+        # input_types[0] can be float even though the input is a quantized tensor
+        # To reliably determine input types, we use Relay's type inference result
+        if _is_quantized_tensor(data, prelude):
```

Comment:
Just one more nit.
Can you move this comment also inside `_is_quantized_tensor` and remove  the below comment from other places

 _# See the comment in adaptive avg pool2d_

Reply:
done